### PR TITLE
feat: seers spinning wheels + doors

### PR DESCRIPTION
**Pull request by DesecratedTree**
## What has been done?
added all wheels to array
made doors and ladder usable

## Has your code been documented?

### CODE REVIEW

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/seers/seers_stairs.plugin.kts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package gg.rsmod.plugins.content.areas.seers
+
+val stairs = arrayOf(
```
Comment (alycii) [inline game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/seers/seers_stairs.plugin.kts:3]: You can (and should) use `listOf` here. Arrays are mutable, whereas Lists are immutable, and the data here is intended to be immutable once loaded.

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/crafting/spinning/spinning_wheel.plugin.kts [hunk 2]
```diff
@@ -4,11 +4,29 @@ package gg.rsmod.plugins.content.skills.crafting.spinning
 val spinningData = SpinningData.values
 val definitions = SpinningData.spinningDefinitions
 
-on_obj_option(obj = Objs.SPINNING_WHEEL_36970, option = "Spin") {
-    val products = spinningData.map { it.product }.toTypedArray()
-    val rawNames = spinningData.map { "(${it.rawName})" }.toTypedArray()
-    player.queue {
-        produceItemBox(*products.toIntArray(), option = SkillDialogueOption.MAKE, title = "Choose how many you wish to cut,<br>then click on the item to begin.", extraNames = rawNames, logic = ::spinItem)
+val wheels = arrayOf(
```
Comment (alycii) [inline game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/crafting/spinning/spinning_wheel.plugin.kts:7]: Same process here, please use `listOf`

**Review by alycii — CHANGES_REQUESTED** [review]
Please resolve the requests :)

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/seers/seers_stairs.plugin.kts (hunk 1 above)
Comment (DesecratedTree) [reply game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/seers/seers_stairs.plugin.kts]: fixed

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/crafting/spinning/spinning_wheel.plugin.kts (hunk 2 above)
Comment (DesecratedTree) [reply game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/crafting/spinning/spinning_wheel.plugin.kts]: fixed

**Review by alycii — APPROVED** [review]

**Review by alycii — APPROVED** [review]

**Pull request merged**